pushy/src/main/java/com/turo/pushy/apns/ApnsClientHandler.java

```diff
@@ -390,7 +387,7 @@ public void onSettingsAckRead(final ChannelHandlerContext ctx) {
 
     @Override
     public void onSettingsRead(final ChannelHandlerContext context, final Http2Settings settings) {
-        log.trace("Received settings from APNs gateway: {}", settings);
+        log.debug("Received settings from APNs gateway: {}", settings);
```

Comment:
We're learning that settings changes are an important part of understanding some performance issues, and I think it makes sense to increase its visibility to `debug` (as a "minor lifecycle event").